### ISSUE TITLE
[Button] Only add `aria-busy` when the component is `loading` (CAL-AMP67)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#13547]
**monday.com sync:** #10821959951

### Violation: 
 Ensure ARIA roles, states, and properties are valid 

### Recommended solution
Only add the `aria-busy` attribute when the component is in a `loading` state, otherwise the attribute should be removed.

- For testing purposes, the following [sample](https://geospatialem.github.io/calcite-samples-2025/5-component-demos/index.html?comp=sheet) was used in the audit. To observe the attribute:
    - Close the `sheet`
    - Inspect the `button`
    - Dive into the shadow DOM to see the native `button` with the added `aria-busy="false"`.

### WCAG Reference: 
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html) 

### Severity: 
 High 

### Media Type: 
 ARIA 

--- 
### Areas for Remediation: 
- Title: CAL-AMP67 Ensure ARIA roles, states, and properties are valid 
- Module: M8 Flow and Sheet 
- Finding ID: [CAL-AMP67](https://esri.hub.essentia11y.com/short-link/1Ez8Ys3gHD_gJqAB) 

**Issue**
There is an "Open Sheet" control with an unnecessary aria-busy and aria-live attribute. 

**User Impact**
Assistive technologies may be unable to parse this content, so users may not be able to perceive the feature(s) or control(s) correctly, or may not be able to operate them.

**Code Reference** 
 ```
<button aria-live="polite" aria-busy="false" aria-label="" class=" button-padding content--slotted icon-start-empty icon-end-empty " href="false" rel="false" target="false" type="button"><!--?lit$029691916$--><!--?lit$029691916$--><!--?lit$029691916$--><span class="content"><slot></slot></span><!--?lit$029691916$--></button>

```

**Steps to Reproduce**
1. Open Chrome DevTools.
2. Locate the element identified in the code reference.
3. Notice that the element's ARIA attributes and/or nesting do not conform to the ARIA specification: https://www.w3.org/TR/wai-aria/ and/or the ARIA in HTML specification: https://www.w3.org/TR/html-aria/
 
**Recommendation**
Ensure ARIA roles, states, and properties are valid. 

In this case, the developer should remove aria-busy and aria-live from the ```<button>``` element.

**Compliant Code Example**
```
<img src="oldbarn.png" alt="Old barn near Jenny Lake" aria-details="imgDetails">
...
<p id="imgDetails">
An old barn with a tin roof and weathered siding stands bathed in morning sunlight near Jenny Lake with the Wyoming Rocky Mountains in the background.
</p>
```

 **Issue Image** 
 https://desktopqa/aic_lap/Calcite/Calcite2025/CAL-AMP67.png 
 

 --- 
### Additional Resources: 
- [LA Platform Report Link](https://esri.hub.essentia11y.com/short-link/1Ez8Ys3gHD_gJqAB) 
- [Esri Accessibility Resources](https://esriis.sharepoint.com/sites/Accessibility)
- Report Source: Product accessibility evaluation conducted on Calcite by Level Access and delivered August, 2025.

**Comment by isaacbraun** [comment]
Verified on `5.2.0-next.71` via https://codepen.io/isaacbraun/pen/GgWmryq that the `aria-busy` attribute is only present (and set to `"true"`) on Button when the `loading` property is also `true`.
